### DISCUSSION TITLE
set_GPIO not working

### DISCUSSION THREAD

**Discussion by Atriredeon** [sle118/squeezelite-esp32#364] [Q&A] [answered]
Hello everyone, i'm trying to set gpio16 high (3.3V) when the ESP32 is online and running and not in sleep. I hooked a button to switch between sleep and running mode. I would like to do that so that I can switch some button lighting on when the player is on and off when I give a ACTRLS_POWER command via the switch. The only problem I have is that when I set the "set_GPIO" value in the NVS Editor to '16=power' (without the quotes), and then commit and reboot. The software will stay in a boot loop and shows only SqueezeESP32 and it won't boot anymore. I need to reflash the ESP32 and start all over again. Is there someone who can help me with this? Thanks on forehand.
[ESP32_Squeezelite_PCM5102+SSD1306+Rotary+Buttons.txt](https://github.com/sle118/squeezelite-esp32/files/13467098/ESP32_Squeezelite_PCM5102%2BSSD1306%2BRotary%2BButtons.txt)

**Comment by Renber78** [comment, marked as answer]
Pin 16 is a dedicated pin for SPI0 (PSRAM), you cannot use it. Which ESP-32 do you have? because normally on the Wrover this pin is not accessible and is marked "NC".
Bàv

**Reply by Atriredeon** [reply]
Hello Renber78, thanks for the information. This was the problem. That also explains why the ESP came into a bootloop. It al makes sense now. I used another GPIO which isn't used by any internally connected chip and it now works as a charm. Thanks for the help!

BTW, I use an ESP32-WROVER-IE dev module which has these pins broke out.
